.pipelines/templates/build-steps.yaml

```diff
@@ -20,5 +20,8 @@ steps:
   # - script: pnpm exec prettier . --check
   #   displayName: Check formatting
 
-  - script: pnpm exec lage lint cover --grouped
-    displayName: Build, Lint and Cover
+  - script: pnpm exec lage lint cover extract-api --grouped
+    displayName: Build, Lint, Cover and check for unexpected API changes
+    env:
+      # Fail the build if package uses api-extractor and there are uncommitted API changes
+      TF_BUILD: 1
```

Comment:
Is this needed? It should be predefined (https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#system-variables-devops-services)

Reply:
I am not sure, I copied it from presentation repo

Reply:
Presentation repo uses Github actions instead of Azure DevOps. So it should not be necessary in here,

Reply:
Let me check if this is the case

Reply:
Yes it works, thanks